src/components/DataTable.js

```diff
@@ -1,40 +1,58 @@
 import React, { Fragment } from 'react';
 import { FormattedMessage, injectIntl } from 'react-intl';
-import { Input, Table, Spin, Alert } from 'antd';
+import { Input, Table, Spin, Alert, Row, Col, Button } from 'antd';
+import { withRouter } from 'react-router-dom';
+import { connect } from 'react-redux';
 
 const Search = Input.Search;
 
-const DataTable = (props) => {
-  const { searchable, updateQuery, fetchData, data, query, loading, error, columns } = props;
+const DataTable = props => {
+  const { searchable, updateQuery, fetchData, data, query, loading, error, columns, noHeader, user } = props;
```

Comment:
I'm not sure when we need to not have a count in the header? I see that it is used to disable it for "published datasets for an organisation". In that example however there can be thousands. I think the header should stay in this case. Could you please explain more about when you think it shouldn't be there?

Reply:
Actually, I decided to hide header on the pages of Published Dataset, Hosted Dataset and Installations within Organization. Now I see that I was wrong in my conclusion.